src/main/java/com/cloudbees/plugins/credentials/CredentialsSelectHelper.java

```diff
@@ -121,6 +125,26 @@ public ModelObject resolveContext(Object context) {
         return context instanceof ModelObject mo ? mo : CredentialsDescriptor.findContextInPath(ModelObject.class);
     }
 
+    /**
+     * @return modifiable store actions for the context provided.
+     */
+    @Restricted(NoExternalUse.class)
+    public Map<String, List<CredentialsStoreAction.DomainWrapper>> getModifiableStoreActions(ModelObject context) {
+        return StreamSupport.stream(CredentialsProvider.lookupStores(context).spliterator(), false)
+                .filter(s -> s.hasPermission(CredentialsProvider.CREATE))
+                .map(CredentialsStore::getStoreAction)
```

Comment:
something weird with the global domain:

<img width="749" height="277" alt="image" src="https://github.com/user-attachments/assets/1f122d95-76df-4011-bc02-52bc117e709b" />

> Failed to load resource: the server responded with a status of 404 (Not Found)

URL is:

```
http://localhost:8099/jenkins/manage/credentials/undefined?
```

Guessing the context change broke global

Reply:
weirdly the `data-url` looks right:

```
/jenkins/manage/credentials/store/system/domain/_/dialog
```

Reply:
Sorted in https://github.com/jenkinsci/credentials-plugin/pull/992/commits/0fcbe92b994c688874e2ed38b3a97a38e5909789